projects/storefrontstyles/scss/components/content/navigation/_group-skipper.scss

```diff
@@ -0,0 +1,26 @@
+%cx-group-skipper {
+  a {
```

Comment:
please follow up with @LTiger14 or @developpeurweb about CSS. we might want to make it more configurable by using scss vars or css vars here